examples/test.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-env mocha */
```

Comment:
I want to be able to run it, immediately. You can't do that if it is stuck in a README.